Error on startup

```
[19:07:36] [Server thread/INFO]: [CreeperHeal2] Enabling CreeperHeal2 v2.0.0
[19:07:36] [Server thread/ERROR]: Error occurred while enabling CreeperHeal2 v2.0.0 (Is it up to date?)
java.lang.NoClassDefFoundError: com/massivecraft/factions/FactionsPlugin
	at net.pdevita.creeperheal2.compatibility.Factions.setPluginReference(Factions.kt:23) ~[?:?]
	at net.pdevita.creeperheal2.compatibility.FactionsProxy.setPluginReference(Factions.kt) ~[?:?]
	at net.pdevita.creeperheal2.compatibility.CompatibilityManager.loadCompatibilityPlugins(CompatibilityManager.kt:20) ~[?:?]
	at net.pdevita.creeperheal2.CreeperHeal2.onEnable(CreeperHeal2.kt:42) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:420) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:467) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:381) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:474) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:290) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at java.lang.Thread.run(Thread.java:830) [?:?]
Caused by: java.lang.ClassNotFoundException: com.massivecraft.factions.FactionsPlugin
	at java.net.URLClassLoader.findClass(URLClassLoader.java:436) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:139) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:85) ~[patched_1.14.4.jar:git-Paper-"6f03c0a5"]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
	... 13 more
[19:07:36] [Server thread/INFO]: [CreeperHeal2] Disabling CreeperHeal2 v2.0.0
```

--- a/src/main/kotlin/net/pdevita/creeperheal2/compatibility/CompatibilityManager.kt
+++ b/src/main/kotlin/net/pdevita/creeperheal2/compatibility/CompatibilityManager.kt
@@ -17,9 +17,13 @@ class CompatibilityManager(val plugin: CreeperHeal2) {
             val otherPlugin = Bukkit.getPluginManager().getPlugin(compatibilityPlugin.pluginName)
             if (otherPlugin != null) {
                 plugin.debugLogger("Loading compatibility for plugin ${compatibilityPlugin.pluginName}")
-                compatibilityPlugin.setPluginReference(otherPlugin)
-                compatibilityPlugin.setCreeperHealReference(plugin)
-                pluginList.add(compatibilityPlugin)
+                try {
+                    compatibilityPlugin.setPluginReference(otherPlugin)
+                    compatibilityPlugin.setCreeperHealReference(plugin)
+                    pluginList.add(compatibilityPlugin)
+                } catch (e: NoClassDefFoundError) {
+                    continue
+                }
             }
         }
     }

